DOCUMENTATION.md

```diff
@@ -217,6 +219,153 @@ uint32_t secure_sum(uint32_t op1, uint32_t op2)
     return secure_gateway(my_box_name, __secure_sum, op1, op2)
 }
 ```
+---
+```C
+UVISOR_BOX_NAMESPACE(static const char const namespace[])
+```
+
+<table>
+  <tr>
+    <td>Description</td>
+    <td colspan="2"><p>Specify the namespace for a box.</p>
+
+      <p>The namespace of the box must be a null-terminated string no longer
```

Comment:
Each paragraph should be a long uninterrupted line (no automatic line break). While I don't like this very much, apparently it is a requirement for the export process (from md to anything else — these docs are used somewhere else). (This is the latest info I got).

Could you please change it? It would be for consistency. In the future we might revert to explicit line breaks, which I'd prefer.